src/connections/ProjectPitch.ts

```diff
@@ -0,0 +1,132 @@
+import { createSelector } from "reselect";
+
+import { EntityIndexFilterKey } from "@/connections/Entity";
+import {
+  adminProjectPitchesIndex,
+  EntityIndexQueryParams,
+  projectPitchesGetUUIDIndex
+} from "@/generated/v3/entityService/entityServiceComponents";
+import { ProjectPitchDto } from "@/generated/v3/entityService/entityServiceSchemas";
+import {
+  adminProjectPitchesIndexFetchFailed,
+  adminProjectPitchesIndexIndexMeta,
+  projectPitchesGetUUIDIndexFetchFailed,
+  projectPitchesGetUUIDIndexIsFetching
+} from "@/generated/v3/entityService/entityServiceSelectors";
+import { getStableQuery } from "@/generated/v3/utils";
+import ApiSlice, { ApiDataStore, PendingErrorState, StoreResourceMap } from "@/store/apiSlice";
+import { Connection } from "@/types/connection";
+import { connectionLoader } from "@/utils/connectionShortcuts";
+import { selectorCache } from "@/utils/selectorCache";
+
+const projectPitchesSelector = () => (store: ApiDataStore) =>
```

Comment:
I don't see any reason to have this intermediate function. This selector can be
```
const projectPitchesSelector = ({ projectPitches }: ApiDataStore) => projectPitches;
```
The in the selector below, you'd use `projectPitchesSelector` instead of `projectPitchesSelector()`